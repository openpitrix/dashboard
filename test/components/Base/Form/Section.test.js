import Setion from 'components/Base/Form/section';

describe('Base/Form Setion', () => {
  it('basic render', () => {
    const wrapper = render(<Setion />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
