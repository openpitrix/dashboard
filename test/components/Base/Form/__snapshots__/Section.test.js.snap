// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base/Form Setion basic render 1`] = `
<div
  class="section"
>
  <p
    class="sectionTitle"
  />
</div>
`;
